### ISSUE TITLE
Do scaffold gap/N characters need to be removed

### ISSUE THREAD

**Issue by mattheatley** [DerrickWood/kraken2#555]
hi,

just curious whether these characters need to be removed if using a scaffold genome? i'm assuming they are not an issue with the complete genome / chromosomes from refseq but i'm using a custom database using genbank scaffold fastas. do these characters need to be removed before adding the sequences to the library/database or can kraken handle them?

**Comment by jenniferlu717** [comment]
@mattheatley no, you do not need to remove these sequences. Kraken will ignore any kmers including those characters.
